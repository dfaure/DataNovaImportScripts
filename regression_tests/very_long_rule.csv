#Identifiant;Libellé_du_site;Date_calendrier;Plage_horaire_1;Plage_horaire_2;Plage_horaire_3;Plage_horaire_4;Plage_horaire_5;Plage_horaire_6;Plage_horaire_7;Heure_limite_dépôt_Courrier;Heure_limite_dépôt_Chrono;Heure_limite_dépôt_Colis;Libellé_de_la_fermeture;Bp_Plage_horaire_1;Bp_Plage_horaire_2;Bp_Plage_horaire_3;Bp_Plage_horaire_4;Bp_Plage_horaire_5;Bp_Plage_horaire_6;Bp_Plage_horaire_7;Bp_Libellé_de_la_fermeture
88064A;BEC DE MORTAGNE AP;2020-12-14;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-09;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-12;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-23;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-10;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-25;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-07;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-26;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-06;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-22;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-27;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-12;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-13;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-24;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-28;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-17;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-18;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-15;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-19;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-14;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-02;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-30;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-03;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-04;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-01;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-29;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-05;09:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-08;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-17;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-09;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-18;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-10;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-16;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-20;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-16;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-07;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-04;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-21;09:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-28;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-15;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-11-11;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-12;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-23;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-10;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-20;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-28;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-11;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-26;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-29;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-08;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-16;09:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-15;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-30;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-18;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-19;09:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-29;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-11;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-06;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-22;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-05;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-09;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-02;09:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-13;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-27;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-21;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-02-06;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-25;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-13;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-19;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-01;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-20;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-03;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-31;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-22;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-14;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-21;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-26;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-25;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-02-01;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-23;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2020-12-24;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-02-02;08:00-11:30;14:00-17:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-02-04;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-02-05;08:00-11:30;14:00-18:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-27;FERME;;;;;;;;;;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-02-08;08:00-11:30;14:00-16:00;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-02-03;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
88064A;BEC DE MORTAGNE AP;2021-01-30;09:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
