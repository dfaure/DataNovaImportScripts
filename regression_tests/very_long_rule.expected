88064A|BEC DE MORTAGNE AP|Mo,Th 08:00-11:30,14:00-16:00;Tu 08:00-11:30,14:00-17:00;week 01-53/2 We 08:00-12:00;2020 week 01-53/2 We off;2020 week 02-53/2 We 08:00-12:00;Fr 08:00-11:30,14:00-18:00;week 02-53/2 Sa 09:00-12:00;2020 week 02-53/2 Sa off;2020 week 01-53/2 Sa 09:00-12:00
